#! /bin/sh /usr/share/dpatch/dpatch-run
## 45_check_http_sni_optional.dpatch
## From: Ton Voon <ton.voon@opsera.com>
## Date: Wed, 7 Jul 2010 11:03:24 +0100
## Subject: [PATCH] Fix examples in check_disk, where it implied was possible to suffix unit values to warn/crit parameters
## X-Git-Url: http://nagiosplug.git.sourceforge.net/git/gitweb.cgi?p=nagiosplug/nagiosplug;a=patch;h=02f9207ebe3a5ae5b35b4b69be1f3bf95ab5a220
##
## DP: Fix examples in check_disk

@DPATCH@

diff --git a/plugins/check_disk.c b/plugins/check_disk.c
index 4913625..851d800 100644
--- a/plugins/check_disk.c
+++ b/plugins/check_disk.c
@@ -963,10 +963,10 @@ print_help (void)
   printf ("%s\n", _("Examples:"));
   printf (" %s\n", "check_disk -w 10% -c 5% -p /tmp -p /var -C -w 100000 -c 50000 -p /");
   printf ("    %s\n", _("Checks /tmp and /var at 10% and 5%, and / at 100MB and 50MB"));
-  printf (" %s\n", "check_disk -w 100M -c 50M -C -w 1000M -c 500M -g sidDATA -r '^/oracle/SID/data.*$'");
+  printf (" %s\n", "check_disk -w 100 -c 50 -C -w 1000 -c 500 -g sidDATA -r '^/oracle/SID/data.*$'");
   printf ("    %s\n", _("Checks all filesystems not matching -r at 100M and 50M. The fs matching the -r regex"));
   printf ("    %s\n", _("are grouped which means the freespace thresholds are applied to all disks together"));
-  printf (" %s\n", "check_disk -w 100M -c 50M -C -w 1000M -c 500M -p /foo -C -w 5% -c 3% -p /bar");
+  printf (" %s\n", "check_disk -w 100 -c 50 -C -w 1000 -c 500 -p /foo -C -w 5% -c 3% -p /bar");
   printf ("    %s\n", _("Checks /foo for 1000M/500M and /bar for 5/3%. All remaining volumes use 100M/50M"));
 
   printf (UT_SUPPORT);

